python/tests/test_util.py

```diff
@@ -0,0 +1,103 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Tskit Developers
+# Copyright (c) 2017 University of Oxford
```

Comment:
Can delete this line, as this file is new.

Reply:
You mean just the 2017 line?

Reply:
Yes, just the 2017 line.

Reply:
Maybe strictly this should be "Copyright (c) 2019 Tskit Developers" as well, since it's a new file.

Reply:
Doesn't matter, ignore the last comments (didn't see it had been resolved).